packages/c/cf-media-manager.json

```
@@ -0,0 +1,37 @@
+{
+  "name": "cf-media-manager",
+  "filename": "cf-media-manager-min.js",
```

```suggestion
  "filename": "browser/media-manager.iife.min.js",
```
Looking at the files you've included, I think this should be this?